BUILD.bazel

```diff
@@ -85,44 +45,20 @@ cc_binary(
         "log",
         "sds",
         "stb_ds",
-        ":libgccjit",
-    ],
+    ] + select({
+        "@bazel_tools//src/conditions:darwin_x86_64": ["@darwin-amd64//:llvm"],
+        "@bazel_tools//src/conditions:linux_aarch64": ["@linux-aarch64//:llvm"],
+        "@bazel_tools//src/conditions:linux_x86_64": ["@linux-amd64//:llvm"],
+        "@bazel_tools//src/conditions:linux_riscv64": ["@linux-riscv64//:llvm"],
+        "@bazel_tools//src/conditions:windows_x64": ["@windows-amd64//:llvm"],
+    }, no_match_error = "not supported os"),
 )
 
 load("@hedron_compile_commands//:refresh_compile_commands.bzl", "refresh_compile_commands")
 
 refresh_compile_commands(
-    name = "refresh_compile_commands_system_libgccjit",
-    targets = {
-        "//:kapuc": "--define use_system_libgccjit=true",
-    },
-)
-
-refresh_compile_commands(
-    name = "refresh_compile_commands_local_libgccjit",
+    name = "refresh_compile_commands",
     targets = {
         "//:kapuc": "",
     },
 )
```

Comment:
This could be cleaned? maybe we don't need alias anymore so we could just remove this? Not sure about this

Reply:
It seems this is still needed?